apps/system/flash_test/flash_test.c

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2016 Samsung Electronics All Rights Reserved.
+ * Copyright 2016, 2021 Samsung Electronics All Rights Reserved.
```

Comment:
You didn't keep original functionality so that I think we make a new function, not reorganize.
I think we can use 2021 only and can remove nuttx license from the c file.